.NET/Microsoft.Recognizers.Definitions.Common/Spanish/DateTimeDefinitions.cs

```diff
@@ -538,18 +547,14 @@ public static class DateTimeDefinitions
             { @"mitad", 0.5 },
             { @"cuarto", 0.25 }
         };
-      public const string DateTokenPrefix = @"en ";
-      public const string TimeTokenPrefix = @"a las ";
-      public const string TokenBeforeDate = @"el ";
-      public const string TokenBeforeTime = @"a las ";
       public const string UpcomingPrefixRegex = @"((este\s+))";
       public static readonly string NextPrefixRegex = $@"\b({UpcomingPrefixRegex}?pr[oó]xim[oa]s?|siguiente|que\s+viene)\b";
       public const string PastPrefixRegex = @"((este\s+))";
       public static readonly string PreviousPrefixRegex = $@"\b({PastPrefixRegex}?pasad[oa](?!(\s+el)?\s+medio\s*d[ií]a)|[uú]ltim[oa])\b";
       public const string ThisPrefixRegex = @"(est?[ea])\b";
-      public static readonly string ThisRegex = $@"\b((este\s*){WeekDayRegex})|({WeekDayRegex}\s*((de\s+)?esta\s+semana))\b";
-      public static readonly string LastDateRegex = $@"\b({PreviousPrefixRegex}\s+{WeekDayRegex})|(este\s+)?({WeekDayRegex}\s+([uú]ltimo|pasado|anterior))|({WeekDayRegex}(\s+((de\s+)?(esta|la)\s+([uú]ltima\s+)?semana)))\b";
-      public static readonly string NextDateRegex = $@"\b(({NextPrefixRegex}\s+){WeekDayRegex})|(este\s+)?({WeekDayRegex}\s+(pr[oó]ximo|siguiente|que\s+viene))|({WeekDayRegex}(\s+(de\s+)?(la\s+)?(pr[oó]xima|siguiente)(\s*semana)))\b";
+      public static readonly string ThisRegex = $@"\b((est[ae]\s*)(semana\s+(el\s+)?)?{WeekDayRegex})|({WeekDayRegex}\s*((de\s+)?esta\s+semana))\b";
```

Comment:
Does English handle "week" in these situation in a similar way?

Reply:
yes, the English regex has the same structure.